packages/web/components/indicators/pill.tsx

```diff
@@ -0,0 +1,7 @@
+import React, { FC } from "react";
+
+export const Pill: FC<{ children: React.ReactNode }> = ({ children }) => (
+  <div className="inline-block flex w-16 items-center justify-center rounded-full bg-gradient-positive px-2 py-1 uppercase text-black">
+    {children}
+  </div>
```

Comment:
This pill doesn't appear to be the same size as figma.

I think the y padding is 2 px too large on top & bottom.